kubernetes/template/DeploymentTemplates/script.sh

```diff
@@ -110,13 +110,27 @@ ensure_certificates()
 # Download msazurestackworkloads' AKSe fork and move relevant files to the working directory
 download_akse()
 {
-    AKSE_ZIP_NAME="aks-engine-$AKSE_RELEASE_VERSION-linux-amd64"
-    AKSE_ZIP_URL="$AKSE_BASE_URL/$AKSE_RELEASE_VERSION/$AKSE_ZIP_NAME.tar.gz"
-    curl --retry 5 --retry-delay 10 --max-time 60 -L -s -f -O $AKSE_ZIP_URL || exit $ERR_AKSE_DOWNLOAD
-    
-    mkdir -p ./bin
-    tar -xf $AKSE_ZIP_NAME.tar.gz
-    cp ./$AKSE_ZIP_NAME/aks-engine ./bin
+    if [ ! $DISCONNECTED_AKS_ENGINE_URL ]
```

Comment:
can we make this so we have one execution flow instead of two?

Reply:
I do not understand what you mean by this

Reply:
I am saying that both code blocks of the if-else statement look really similar, is it possible to have a single execution path?